stories/Molecules/Imagecaption/imagecaption.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../assets/scss/variables";
+@import "../../assets/scss/breakpoints";
+
+/* image caption */
+.image-caption {
+    display: block;
+    border-bottom: 2px solid $color-black;
+    @include devicebreak(small) {
+        display: flex;
+        padding-bottom: 25px;
+    }
+    .image__caption__para {
+        padding-right: 0px;
+        padding-bottom: 15px;
+        margin-bottom: 15px;
+        line-height: 19.25px;
+        color: $color-gray-600;
+        font-size: $font-size-14;
+        border-bottom: 1px solid $color-gray-500;
+        @include devicebreak(small) {
+            margin: 0;
+            border: 0;
+            padding-bottom: 0;
+            line-height: 22px;
+            padding-right: 20px;
+            font-size: $font-size-16;
+        }
+    }
+}
+
+.image--author {
+    flex-shrink: 0;
+    position: relative;
+    margin-bottom: 15px;
+    @include devicebreak(small) {
+        margin-bottom: 0;
+        padding-left: 40px;
+    }
+    &:before {
+        content: "";
+        top: 0;
+        left: 0;
+        width: 0px;
+        height: 45px;
```

Comment:
@Shubham-Sharma-77 Why we are fixing the height, can't we manage without fixing the height